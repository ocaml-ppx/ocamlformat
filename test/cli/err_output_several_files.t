  $ echo 'let x = 1' > a.ml
  $ echo 'let x = 2' > b.ml

  $ ocamlformat --output x.ml a.ml b.ml
  ocamlformat: Must specify exactly one input file without --inplace or --check
  [1]
