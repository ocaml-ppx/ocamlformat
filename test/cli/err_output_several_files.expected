ocamlformat: Must specify exactly one input file without --inplace or --check
